exts/cesium.omniverse/cesium/omniverse/ui/debug_window.py

```diff
@@ -60,10 +60,8 @@ def print_fabric_stage():
             self._cesium_message_field.set_value(fabric_stage)
 
         with ui.VStack():
-            ui.Button("Remove all Tilesets", clicked_fn=lambda: remove_all_tilesets())
-            ui.Button("Reload all Tilesets", clicked_fn=lambda: reload_all_tilesets())
-            ui.Button("Print Fabric stage", clicked_fn=lambda: print_fabric_stage())
-            with ui.VStack():
-                self._cesium_message_field = ui.SimpleStringModel("")
-                ui.StringField(self._cesium_message_field, multiline=True, read_only=True)
+            ui.Button("Remove all Tilesets", height=20, clicked_fn=lambda: remove_all_tilesets())
+            ui.Button("Reload all Tilesets", height=20, clicked_fn=lambda: reload_all_tilesets())
+            ui.Button("Print Fabric stage", height=20, clicked_fn=lambda: print_fabric_stage())
```

Comment:
Can you go ahead and fix all of these? We can just pass in the functions as higher-order and this should work. You may need to stub out some parameters using `_` but I don't think you will in this case.